src/java/org/apache/cassandra/db/view/ViewBuilder.java

```diff
@@ -209,4 +206,16 @@ public boolean isGlobal()
     {
         return false;
     }
+
+    public void waitForCompletion()
```

Comment:
I think this method can be `protected` and not `public`?

Reply:
Right, it can even have default/package visibility since we don't extend the class. Done [here](https://github.com/apache/cassandra/pull/980/commits/0f0edd0bb916a9864a25f698187c53aab09891bf).